libs/sr/page-team/src/TeammateDisplay.tsx

```
@@ -14,37 +20,167 @@ import {
+const SECTION_SPACING_PX = 33
+const SectionNumContext = createContext(0)
 export default function TeammateDisplay() {
-  const {
-    teammateDatum: { characterKey, buildType },
-  } = useTeamContext()
+  const { presetIndex } = useContext(PresetContext)
+  const { team } = useTeamContext()
+  const sections: Array<[key: string, title: ReactNode, content: ReactNode]> =
+    useMemo(() => {
+      const frame = team.frames[presetIndex]
+      if (!frame) return [['char', 'Character', <CharacterSection />]]
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)